src/docs/product/releases/suspect-commits/index.mdx

```diff
@@ -82,31 +86,111 @@ The following example associates commits (or refs) between `from` and `to` with
 sentry-cli releases set-commits --commit "my-repo@from..to" $VERSION
 ```
 
+Alternatively, you can use the `--local` flag to enable this behavior by default.
```

Comment:
It's not clear what behavior this is talking about. (I get that you're just copying from the sentry-cli docs, but I think it needs to be clarified. It's right after a paragraph about specifying which commits to use, but the options seems to be about where to get the commit data, not which commits to get data for.)

@kamilogorek - This option:

https://github.com/getsentry/sentry-cli/blob/7353fd8fc5611221fc318c325ed2b5bb8ec17c7a/src/commands/releases/set_commits.rs#L33-L39

I get that it pulls from the local repo, but _what_ does it pull from that repo? Which commits are included?

Reply:
It will fetch all commits between the `HEAD` (the latest commit available locally) and the most recent one that was used for the previous Sentry release.

Reply:
So how is it different than `--auto`, then?